util/measures/2018/pi-measure-relations.json

```diff
@@ -1,42 +1,52 @@
-  "PI_CCTPE_2" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_TRANS_SM_1"]},
-  "PI_CCTPE_1" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_TRANS_PEA_2"]},
-  "PI_HIE_3" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_TRANS_MR_1"]},
-  "PI_CCTPE_3" : {"reportingCategory":"performanceBonus", "substitutes": []},
+  "PI_TRANS_PHCDRR_1": { "reportingCategory": "performanceBonus", "substitutes": ["PI_PHCDRR_1"]},
+  "PI_TRANS_PSE_1": { "reportingCategory": "performanceBonus", "substitutes": ["PI_PEA_2"]},
+  "PI_TRANS_SM_1": { "reportingCategory": "performanceBonus", "substitutes": ["PI_CCTPE_2"]},
+  "PI_TRANS_PEA_2": { "reportingCategory": "performanceBonus", "substitutes": ["PI_CCTPE_1"]},
+  "PI_TRANS_MR_1": { "reportingCategory": "performanceBonus", "substitutes": ["PI_HIE_3"]},
 
-  "PI_TRANS_PHCDRR_1" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_PHCDRR_1"]},
-  "PI_TRANS_PSE_1" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_PEA_2"]},
-  "PI_TRANS_SM_1" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_CCTPE_2"]},
-  "PI_TRANS_PEA_2" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_CCTPE_1"]},
-  "PI_TRANS_MR_1" : {"reportingCategory":"performanceBonus", "substitutes": ["PI_HIE_3"]},
+  "PI_PHCDRR_2": { "reportingCategory": "registryBonus", "substitutes": ["PI_TRANS_PHCDRR_2"]},
+  "PI_PHCDRR_5": { "reportingCategory": "registryBonus", "substitutes": ["PI_TRANS_PHCDRR_3"]},
+  "PI_PHCDRR_3": { "reportingCategory": "registryBonus", "substitutes": []},
+  "PI_PHCDRR_4": { "reportingCategory": "registryBonus", "substitutes": []},
 
-  "PI_PHCDRR_2" : {"reportingCategory":"registryBonus", "substitutes": ["PI_TRANS_PHCDRR_2"]},
-  "PI_PHCDRR_5" : {"reportingCategory":"registryBonus", "substitutes": ["PI_TRANS_PHCDRR_3"]},
-  "PI_PHCDRR_3" : {"reportingCategory":"registryBonus", "substitutes": []},
-  "PI_PHCDRR_4" : {"reportingCategory":"registryBonus", "substitutes": []},
-  "PI_TRANS_PHCDRR_2" : {"reportingCategory":"registryBonus", "substitutes": ["PI_PHCDRR_2"]},
-  "PI_TRANS_PHCDRR_3" : {"reportingCategory":"registryBonus", "substitutes": ["PI_PHCDRR_5"]},
+  "PI_PHCDRR_5_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
+  "PI_PHCDRR_3_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
+  "PI_PHCDRR_1_MULTI": { "reportingCategory": "performanceBonus", "substitutes": []},
+  "PI_PHCDRR_4_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
+  "PI_PHCDRR_2_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
 
-  "PI_IACEHRT_1": {"reportingCategory": "cehrtBonus", "substitutes": []}
-}
+  "PI_TRANS_PHCDRR_1_MULTI": { "reportingCategory": "performanceBonus", "substitutes": []},
+  "PI_TRANS_PHCDRR_3": { "reportingCategory": "registryBonus", "substitutes": ["PI_PHCDRR_5"]},
+  "PI_TRANS_PHCDRR_3_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
```

Comment:
For the new measures:
```
PI_PHCDRR_1_MULTI
PI_PHCDRR_2_MULTI
PI_PHCDRR_3_MULTI
PI_PHCDRR_4_MULTI
PI_PHCDRR_5_MULTI

PI_TRANS_PHCDRR_1_MULTI
PI_TRANS_PHCDRR_2_MULTI
PI_TRANS_PHCDRR_3_MULTI
```

I'm wondering if the same substitute relationships needs to exist for the similar named measure.  One example, does `PI_PHCDRR_5_MULTI` need to be considered a substitute of `PI_TRANS_PHCDRR_3_MULTI` in the same way `PI_PHCDRR_5` is considered a substitute of `PI_TRANS_PHCDRR_3`? @sarahnw @NateTheGreatt 

Same situation for `PI_TRANS_PHCDRR_2_MULTI`, are there substitutes similar to `PI_TRANS_PHCDRR_2`?

Reply:
good question. I don't totally know what the substitute field is for, if that's for scoring or policy. let me check with PI policy.

Reply:
@rockykitamura policy says yes they should

Reply:
Great.  @NateTheGreatt will make the required updates.

Reply:
@sarahnw @rockykitamura none of the aformentioned measureIds have those substitute values in the provided `pi-measures.json` file in the ticket https://jira.cms.gov/browse/QPPSF-2762